### PR TITLE
Add translation in 🇫🇷 for Camp2 students

### PR DESCRIPTION
**Pull request by tgautier**
I think this is a nice change :wink:

### CODE REVIEW

**Comment by tgautier** [comment]
Hey, what do you think about those changes?

File: demo.txt [hunk 1]
```diff
@@ -1 +1,3 @@
-Hello Camp2!
+Hello Camp2 students!
+
+Salut les étudiants de Camp2!
```
Comment (RomSarto) [inline demo.txt]: Salut les étudiants de Camp2 !

**Review by RomSarto — CHANGES_REQUESTED** [review]
In French, we add a space before a `!`

File: demo.txt (hunk 1 above)
Comment (Fenntasy) [inline demo.txt]: Les étudiants et les étudiantes.

**Review by Fenntasy — CHANGES_REQUESTED** [review]
Can you change this line?

File: demo.txt [hunk 2]
```diff
@@ -1 +1,3 @@
-Hello Camp2!
+Hello Camp2 students!
+
```
Comment (Fenntasy) [inline demo.txt:2]: comment

File: demo.txt (hunk 1 above)
Comment (Florent1972) [reply demo.txt]: Les étudiantes et les étudiants.

**Review by Fenntasy — APPROVED** [review]
👍

**Pull request merged**